adapters/info_test.go

```diff
@@ -69,57 +72,143 @@ func TestImpFiltering(t *testing.T) {
 	}
 
 	constrained := adapters.EnforceBidderInfo(bidder, info)
-	_, errs := constrained.MakeRequests(&openrtb.BidRequest{
-		Imp: []openrtb.Imp{
-			{
-				ID:    "imp-1",
-				Video: &openrtb.Video{},
+
+	// Define test cases
+	type testOut struct {
```

Comment:
Does this really have to be a separate struct? Why not add `expectedErrors` and `expectedImps` to the `testCases` struct?

Reply:
Modified